posts/napari-cosmx-basics/using-napari-cosmx.qmd

```diff
@@ -0,0 +1,501 @@
+the plugin. 
+
+# Working with layers {#sec-working-with-layers}
+
+When we open a slide with `napari-cosmx`, by default there will be a few napari
+layers visible (Initial View tab; @fig-initial). These include `FOV labels` and `Segmentation`. Clicking the eye
+icon next to a layer will change its visibility. Let's turn off those layers 
+for a moment and visualize the cell types from the `RNA_nbclust_clusters` column 
+(Cell Types tab; @fig-cell-types). We can also color cells by their spatialClusteringAssignments values (Niches tab; @fig-niches). 
+In the `Color Cells` widget, we can control which cell types or niches
+we would like to view. When we activate the `Metadata` layer, hovering over
+a given cell will display the metadata associated with it as a ribbon at the bottom
+of the application. To view the IF channels, use
+the `Morphology Images` widget, In @fig-IF (IF Channels tab) I turned off the visibility of the cell
+types, added GFAP in red and DNA in blue, and zoomed into the hippocampus. When
+I click on a layer, it becomes the activate layer and I can use the `layer controls` widget
+to adjust attributes to that layer such as contrast
+limits, gamma, layer blending, and more. Finally, we can view raw transcripts 
+(or proteins). Simply select the target and the color and click `Add layer`. In 
+@fig-tx (Transcripts tab), I zoomed in on a section of the cortex and plotted _Calb1_. 
+
+Like other programs that use layers, napari allows the layers to be moved up/down
+and to blend (not shown below).
+
+::: {.panel-tabset}
+
+## Initial View
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-initial"
+#| fig-cap: "The initial view of the tissue shows the location of FOVs and cell boundary layers."
+
+knitr::include_graphics("./figures/fig-initial.png")
+```
+
+## Cell Types
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-cell-types"
+#| fig-cap: "Same extent as @fig-initial and displaying the cell type from the 'RNA_nbclust_clusters'."
+
+knitr::include_graphics("./figures/fig-cell-types-short.png")
+```
+
+## Niches
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-niches"
+#| fig-cap: "Cells colored by niche"
+
+knitr::include_graphics("./figures/fig-niches.png")
+```
+
+## IF Channels
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-IF"
+#| fig-cap: "Hippocampal region of tissue with GFAP (red) and DNA (blue)."
+
+knitr::include_graphics("./figures/fig-IF.png")
+```
+
+## Transcripts
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-tx"
+#| fig-cap: "Expression of _Calb1_ in cortex layer I."
+
+knitr::include_graphics("./figures/fig-transcripts.png")
+```
+
+:::
+
+# Scripting tips-n-tricks {#sec-scripting}
+
+This section is for advanced users who want finer control of the aesthetics.
+
+Most of the items we've covered can also be accessed through various methods in the `gem` object
+that can found loaded in the `>_` `ipython` interpreter. You may have
```

Comment:
It would be easier to understand if first highlight the location of `>_` console button in GUI and then mention one can interact with the `gem` object to manipulate the viewer and data via scripting in console. After that, you can list the various examples and tricks in details. 
The first example you showed is to take screenshot at specific camera center and zoom for a specific combination of visible layers. It would be good to list it as one of the itermized topic under section 4.